src/index.js

```diff
@@ -302,6 +313,43 @@ class PublicationClient extends EventEmitter {
     return subscription;
   }
 
+  /**
+   * Subscribe to the publication via callback parameters. If pressented expected to be called
+   * on receiving the events.
+   *
+   * @param {function} onReconnectedEvent The function that should be called when receiving reconnection.
+   * @param {function} onAddedEvent The function that should be called when receiving added updates.
+   * @param {function} onChangedEvent The function that should be called when receiving changed updates.
+   * @param {function} onRemovedEvent The function that should be called when receiving removed updates.
+   */
+  subscribeEventsToCallbacks(onReconnectedEvent, onAddedEvent, onChangedEvent, onRemovedEvent) {
```

Comment:
@bradvogel I will provide once I have this merged. Because it will crash without updating the publication-client version (subscribeEventsToCallbacks doesn't exist yet).

I can only show like this:
<img width="666" alt="Screenshot 2022-08-09 at 19 50 13" src="https://user-images.githubusercontent.com/7041061/183710642-eadf1b5b-d8be-45b3-9fa4-480dd1f90649.png">

Reply:
@PandasProperty Thanks! Perhaps put up a draft PR that uses this just so we can see more about how the `on` event listeners (eg `onAddedEvent`) handle the data?